notebooks/knn.ipynb

```diff
@@ -0,0 +1,399 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "db673eda-86ee-47a7-975d-aa3d36c2f407",
+   "metadata": {},
+   "source": [
+    "# KNN"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eeb78fae-ae08-4b64-8daa-b579af1d9ba6",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "from spark_rapids_ml.knn import NearestNeighbors\n",
+    "from pyspark.sql import DataFrame, Row\n",
+    "from pyspark.sql.functions import col, lit\n",
+    "from pyspark.sql.types import IntegerType, StructType, StructField"
```

Comment:
It seems "lit", "IntegerType", "StructType", and "StructField" do not appear in other places of the notebook. 
May need to delete them if not used. 

It seems "DataFrame" and "Row" are also not used.

Reply:
Good catch, removed.